src/chrome/content/rules/SchuelerVZ.xml

```diff
@@ -1,7 +1,6 @@
-<ruleset name="SchuelerVZ.net" default_off="Certificate mismatch">
-<target host="schuelervz.net" />
-  <target host="www.schuelervz.net" />
+<ruleset name="SchuelerVZ.net">
+	<target host="schuelervz.net" />
```

Comment:
This host times out.
See also `static.pe.schuelervz.net` and `blog.schuelervz.net`.